src/MessagingAttachment/MessagingAttachment.less

```diff
@@ -0,0 +1,188 @@
+  padding-bottom: 0.9375rem; // 15px
+  height: @ICON_REGULAR_SIZE;
+  border: @size_4xs solid @neutral_silver;
+  border-radius: @size_2xs;
+}
+
+.MessagingAttachment--IconContainer {
+  height: @ICON_REGULAR_SIZE;
+  .margin--right--s();
+  align-items: center;
+  justify-content: center;
+}
+
+// handle FontAwesome as well, for loading spinner
+.MessagingAttachment--IconContainer img,
+.MessagingAttachment--IconContainer .fa,
+.MessagingAttachment--IconContainer > * {
+  max-height: @ICON_REGULAR_SIZE;
+  max-width: @ICON_REGULAR_SIZE;
+  width: auto;
+  height: auto;
+}
+
+.MessagingAttachment--TextContainer {
+  justify-content: center;
+  color: @neutral_dark_gray;
+  max-width: 18.375rem; // 294px
+}
+
+// TODO: remove if unused
+.MessagingAttachment--Title {
+  font-size: @size_m;
+  line-height: @size_l;
+  .text--semi-bold();
+  .text--truncate();
+}
+
+.MessagingAttachment--Subtitle {
+  font-size: 0.875rem; // 14px
+  line-height: @size_m;
+  .text--truncate();
+}
+
+.MessagingAttachment--CloseButton {
+  z-index: 10;
+  position: absolute;
+  .flexbox();
+  top: -0.625rem; // 10px
+  right: -0.625rem; // 10px
+  width: 1.25rem; // 20px
+  height: 1.25rem; // 20px
+  border-radius: 50%;
+  cursor: pointer;
+  background-color: @neutral_medium_gray;
+  color: @WHITE;
+  border: none;
+  padding: 0;
+  border: @size_3xs solid white;
+
+  // TODO: doesn't turn off after click...
+  // maybe this doesn't matter cause clicking will close the box?
+  &:hover,
+  &:focus,
+  &:active {
+    background-color: @neutral_black;
+    border-color: @neutral_black;
+  }
+}
+
+.MessagingAttachment--CloseIcon {
+  font-size: @size_s;
+  margin: auto;
+}
+
+.MessagingAttachment--Container.MessagingAttachment--Short {
+  width: 12.75rem; // 204px
+  cursor: default;
+  height: @ICON_SMALL_SIZE;
+  padding-left: 0.4375rem; // 7px
+  padding-right: 0.4375rem; // 7px
+  padding-top: 0.6875rem; // 11px
+  padding-bottom: 0.6875rem; // 11px
+
+  .MessagingAttachment--IconContainer {
+    height: @ICON_SMALL_SIZE;
+    .margin--right--xs();
+  }
+
+  // handle FontAwesome as well, for loading spinner
+  .MessagingAttachment--IconContainer img,
+  .MessagingAttachment--IconContainer .fa,
+  .MessagingAttachment--IconContainer > * {
+    max-height: @ICON_SMALL_SIZE;
+    max-width: @ICON_SMALL_SIZE;
+  }
+
+  .MessagingAttachment--TextContainer {
+    max-width: 10.25rem; // 164px
+  }
+
+  .MessagingAttachment--Title {
+    line-height: @size_m;
+    font-size: 0.875rem; // 14px
+  }
+
+  .MessagingAttachment--Subtitle {
+    font-size: 0.875rem; // 14px
+    line-height: @size_m;
+    color: @neutral_dark_gray;
+    .text--regular();
+    margin-top: 0;
+  }
+
+  &.MessagingAttachment--IsUploading {
+    width: 12.25rem; // 196px
+    padding-left: 0.6875rem; // 11px
+    padding-right: 0.6875rem; // 11px
+
+    .MessagingAttachment--TextContainer {
+      max-width: 10.375rem; // 166px
+    }
+  }
+}
+
+// TODO: :not() does not work for IE
```

Comment:
I assume this doesn't actually impede functionality in any way? Not ideal then, but we already have some serious styling issues on IE if I'm remembering right, so I think something this small may very well be fine.

Reply:
It only effects the hover states. I think I can remake it without the not though

Reply:
It would just be some redundant css